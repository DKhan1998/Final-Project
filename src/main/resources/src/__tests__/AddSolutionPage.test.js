import React from "react";
import AddSolutionPage from "../Components/Pages/AddSolutionPage"
import render from "react-test-renderer";

//snapshot test
test("it matches the snapshot", () => {
    const tree = render.create(<AddSolutionPage/>).toJSON();
    expect(tree).toMatchSnapshot();
})