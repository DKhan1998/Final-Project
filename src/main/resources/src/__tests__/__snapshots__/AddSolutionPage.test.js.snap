// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it matches the snapshot 1`] = `
<div>
  <h1
    id="pageheader"
  >
     Add Solution
  </h1>
  <div>
    <div
      style={
        Object {
          "display": "none",
        }
      }
    >
      <div
        style={null}
      />
    </div>
    <form
      className=""
      id="ticketForm"
      onReset={[Function]}
      onSubmit={[Function]}
    >
      <div
        className="form-row"
      >
        <div
          className="form-group col"
        >
          <label
            className="form-label"
            htmlFor="formCohort"
          >
            Cohort
          </label>
          <select
            autoComplete="off"
            className="form-control"
            id="formCohort"
            name="cohort"
            onChange={[Function]}
            required={true}
            value=""
          >
            <option>
              Choose...
            </option>
            <option
              value="Azure-Devops"
            >
              Azure Devops - (20OctDevOps1)
            </option>
            <option
              value="Software-BAE"
            >
              Software BAE - (20OCTBAE2)
            </option>
            <option
              value="SDET"
            >
              SDET - (20OCTSDET3)
            </option>
            <option
              value="Cloud-Native-Engineer"
            >
              Cloud Native Engineer - (20SeptCNE)
            </option>
            <option
              value="Scala"
            >
              Scala - (20NovSoftware1)
            </option>
          </select>
        </div>
        <div
          className="form-group col"
        >
          <label
            className="form-label"
            htmlFor="formAuthor"
          >
            Author
          </label>
          <input
            autoComplete="off"
            className="form-control"
            id="formAuthor"
            name="author"
            onChange={[Function]}
            required={true}
            type="text"
            value=""
          />
        </div>
      </div>
      <div
        className="form-group"
      >
        <label
          className="form-label"
          htmlFor="formTitle"
        >
          Issue Title
        </label>
        <input
          autoComplete="off"
          className="form-control"
          id="formTitle"
          name="title"
          onChange={[Function]}
          required={true}
          type="text"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className="form-label"
          htmlFor="formDescription"
        >
          Issue Description
        </label>
        <textarea
          autoComplete="off"
          className="form-control"
          id="formDescription"
          name="description"
          onChange={[Function]}
          required={true}
          rows="4"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className="form-label"
          htmlFor="formSolution"
        >
          Solution
        </label>
        <textarea
          autoComplete="off"
          className="form-control"
          id="formSolution"
          name="solution"
          onChange={[Function]}
          required={true}
          rows="4"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          className="form-label"
          htmlFor="formLinks"
        >
          Links
        </label>
        <input
          autoComplete="off"
          className="form-control"
          id="formLinks"
          name="links"
          onChange={[Function]}
          required={true}
          type="text"
          value=""
        />
      </div>
      <button
        className="btn btn-success"
        disabled={false}
        onClick={[Function]}
        type="submit"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-save fa-w-14 "
          data-icon="save"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 448 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M433.941 129.941l-83.882-83.882A48 48 0 0 0 316.118 32H48C21.49 32 0 53.49 0 80v352c0 26.51 21.49 48 48 48h352c26.51 0 48-21.49 48-48V163.882a48 48 0 0 0-14.059-33.941zM224 416c-35.346 0-64-28.654-64-64 0-35.346 28.654-64 64-64s64 28.654 64 64c0 35.346-28.654 64-64 64zm96-304.52V212c0 6.627-5.373 12-12 12H76c-6.627 0-12-5.373-12-12V108c0-6.627 5.373-12 12-12h228.52c3.183 0 6.235 1.264 8.485 3.515l3.48 3.48A11.996 11.996 0 0 1 320 111.48z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
         SOLVE
      </button>
        
      <button
        className="btn btn-info"
        disabled={false}
        onClick={[Function]}
        type="button"
      >
        <svg
          aria-hidden="true"
          className="svg-inline--fa fa-list fa-w-16 text-white"
          data-icon="list"
          data-prefix="fas"
          focusable="false"
          role="img"
          style={Object {}}
          viewBox="0 0 512 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M80 368H16a16 16 0 0 0-16 16v64a16 16 0 0 0 16 16h64a16 16 0 0 0 16-16v-64a16 16 0 0 0-16-16zm0-320H16A16 16 0 0 0 0 64v64a16 16 0 0 0 16 16h64a16 16 0 0 0 16-16V64a16 16 0 0 0-16-16zm0 160H16a16 16 0 0 0-16 16v64a16 16 0 0 0 16 16h64a16 16 0 0 0 16-16v-64a16 16 0 0 0-16-16zm416 176H176a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h320a16 16 0 0 0 16-16v-32a16 16 0 0 0-16-16zm0-320H176a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h320a16 16 0 0 0 16-16V80a16 16 0 0 0-16-16zm0 160H176a16 16 0 0 0-16 16v32a16 16 0 0 0 16 16h320a16 16 0 0 0 16-16v-32a16 16 0 0 0-16-16z"
            fill="currentColor"
            style={Object {}}
          />
        </svg>
        <span
          className="text-white"
        >
          Ticket List
        </span>
      </button>
    </form>
  </div>
</div>
`;
